### ISSUE TITLE
pvc stopped working in latest release

### ISSUE THREAD

**Issue by garnett** [facebook/chisel#209]
swift4, Xcode9.1:

```
pvc
error: error: use of undeclared identifier 'id'
error: expected expression
None
```

**Comment by garnett** [comment]
According to the https://github.com/facebook/chisel/wiki#pvc it expects a parameter and works if it's given one... I used it before without any to print all controllers though.

**Comment by ThibaultFarnier** [comment]
Also having the issue with Obj-C projects

**Comment by kastiglione** [comment]
@garnett thanks for the report. An argument isn't required. I'll investigate.

**Comment by kastiglione** [comment]
I have a fix in #210. If you're not using the XCTest helper commands, then you could easily test it by editing fblldbbase.py. In there, find `evaluateExpressionValue` and swap the order of `printErrors` and `language`, as shown in this diff. I plan to push out a 1.6.1 fix release tonight.
